target/mips/helper.c

```diff
@@ -1974,9 +1973,13 @@ int cheri_tag_get_many(CPUMIPSState *env, target_ulong vaddr, int reg,
         hwaddr *ret_paddr, uintptr_t pc)
 {
     uint64_t tag;
+    int prot;
     uint8_t *tagblk = cheri_tag_get_block(env, vaddr, MMU_DATA_CAP_LOAD, reg,
                                           CAP_TAG_GET_MANY_SHFT, pc, ret_paddr,
-                                          NULL, &tag);
+                                          NULL, &tag, &prot);
+
+    (void) prot; // XXX?
```

Comment:
I don't think we any code here, but a comment about potentially changing the spec for CLoadTags would be fine.

Reply:
Added.